docs/admin/authorization.md

```diff
@@ -500,3 +500,42 @@ to a remote authorization service.  Authorization modules can implement
 their own caching to reduce the cost of repeated authorization calls with the
 same or similar arguments.  Developers should then consider the interaction
 between caching and revocation of permissions.
+
+
+### Checking API Access
+
+The `subjectaccessreviews.v1beta1.authorization.k8s.io` resource is exposed as a normal
```

Comment:
What exposes it? Kubernetes?

"Kubernetes exposes the `subjectaccessreviews.v1beta1.authorization.k8s.io` resource as a normal resource that allows external access to API authorizer decisions."